### ISSUE TITLE
RE: Lab with fresh TIG stack install on Ubuntu

### ISSUE THREAD

**Issue by Tanzir14** [sttrayno/ASA-Telemetry-Guide#3]
![image](https://user-images.githubusercontent.com/52311812/101276630-3b949e00-377c-11eb-968e-3598e65fa889.png)

Hi Stuart-

This must be an amazing way to monitor the ASA VPN. I have come across this a while ago, and still kind of waiting on the updated lab with a fresh TIG install along with the steps to install all the needed components. I plan to utilize this on a VM running Ubuntu, or (preferably) trying to see how I can have this running on a Windows Server 2019. Will appreciate some assistance. Thanks a lot for this!

Best personal regards,

Tanzir

**Comment by sttrayno** [comment]
Hi Tanzir,

I still haven't found the time to do this, however there are so many guides to installing TIG stack on Linux so it's been a lower priority. Have a look at something like this which should get your VM set up:

https://kifarunix.com/install-and-setup-tig-stack-on-ubuntu-20-04/

Once you have that setup you should be able to get it setup with this guide fairly easily. Drop me a shout if you have any issues, I'll leave this open for a while.

Thanks,
Stuart

**Comment by Tanzir14** [comment]
> Hi Tanzir,
> 
> I still haven't found the time to do this, however there are so many guides to installing TIG stack on Linux so it's been a lower priority. Have a look at something like this which should get your VM set up:
> 
> https://kifarunix.com/install-and-setup-tig-stack-on-ubuntu-20-04/
> 
> Once you have that setup you should be able to get it setup with this guide fairly easily. Drop me a shout if you have any issues, I'll leave this open for a while.
> 
> Thanks,
> Stuart

Thanks a lot, Stuart. Appreciate you getting back on this. I plan to start setting up the VM sometime soon; just don't have an exact time on that, yet. I greatly appreciate you leaving this open for a while. I am sure I'll come across issues and questions, and thus asking for more help! 

For now, I have 2 quick questions:

- Is it possible to grab more information from the AnyConnect sessionsdb? Having a basic chart with currently connected users, their IPs, connection period, would've made this more helpful.

- After I am done with [Install and Setup TIG Stack on Ubuntu 20.04](https://kifarunix.com/install-and-setup-tig-stack-on-ubuntu-20-04/), which part of the [ASA VPN Health Monitoring](https://developer.cisco.com/codeexchange/github/repo/sttrayno/ASA-Telemetry-Guide) - Step 2 do I need to continue on from?

Best personal regards,

Tanzir
